CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+We are a small team and this project is one of many projects. We would love to see, when you help us developing new checks. Here a small how-to.
+
+### How-to
+
+Fork our repository, then clone/pull the forked repo via ABAPgit into your system:
+
+    https://github.com/your-username/code-pal-for-abap.git
+
+Create and test your new Check in your system.
+
+When you are done. Please `stage` --> `commit` --> `push` the files to your fork.
+
+Back to github.com you can create a pull request from your fork to our base repo.
+
+This is the point, where we'll look into your code and approve the merge. Please create the request to merge with our `dev` branch, because our work-flow is from `dev` to `test` to `master`.
```

Comment:
At this point of time, we will verify your code and authorize/approve the merge (if applicable). Please create the request to merge it with our `dev` branch (our release workflow starts from `dev`, then to `test` and finally it reaches the `master` branch.